cms/djangoapps/contentstore/views/organization.py

```diff
@@ -18,6 +20,7 @@ class OrganizationListView(View):
     @method_decorator(login_required)
     def get(self, request, *args, **kwargs):
         """Returns organization list as json."""
-        organizations = get_organizations()
-        org_names_list = [(org["short_name"]) for org in organizations]
+        current_site = get_current_site(request)
+        edly_sub_organization = EdlySubOrganization.objects.filter(studio_site=current_site.id)
```

Comment:
We can backtrack `EdlySubOrganization` from a site (https://github.com/edly-io/edx-platform/pull/71).
Also we need to keep the changes in the existing codebase to minimal in order to avoid any code conflicts in future rebase so I would suggest to create a method `get_edly_organizations` in a `utils.py` file here https://github.com/edly-io/edx-platform/tree/develop-multisite/openedx/features/edly then we can call that method here:
```
 organizations = get_edly_organizations(request)
```
Also django filter (https://docs.djangoproject.com/en/3.0/ref/models/querysets/#filter) returns a queryset not just one object so we need to name variables accordingly.

Reply:
Done